packages/core/test/sdk.wrap.test.ts

```diff
@@ -0,0 +1,97 @@
+import { render } from '@testing-library/react-native';
+import * as React from 'react';
+
+import { wrap } from '../src/js/sdk';
+
+jest.mock('../src/js/touchevents', () => ({
+  __esModule: true,
+  TouchEventBoundary: ({ children }: { children: React.ReactNode }) =>
+    React.createElement('div', { testID: 'touch-boundaryID' }, children),
+}));
+
+jest.mock('../src/js/tracing', () => ({
+  __esModule: true,
+  ReactNativeProfiler: jest.fn(({ children }: { children: React.ReactNode }) =>
+    React.createElement('div', { testID: 'profilerID' }, children),
+  ),
+}));
+
+jest.mock('../src/js/feedback/FeedbackWidgetManager', () => ({
+  __esModule: true,
+  FeedbackWidgetProvider: ({ children }: { children: React.ReactNode }) =>
+    React.createElement('div', { testID: 'feedback-widgetID' }, children),
+}));
+
+import type { ReactNativeWrapperOptions } from 'src/js/options';
+
+import { ReactNativeProfiler } from '../src/js/tracing';
+
+// Test
+describe('wrap', () => {
+  const DummyComponent: React.FC<{ value?: string }> = ({ value }) => React.createElement('div', null, value);
+
+  it('wraps components with Sentry wrappers', () => {
+    const Wrapped = wrap(DummyComponent);
+    const element = React.createElement(Wrapped, { value: 'wrapped' });
+    const renderResult = render(element);
+
+    const { getByTestId } = renderResult;
+    expect(getByTestId('touch-boundaryID')).toBeDefined();
+    expect(getByTestId('profilerID')).toBeTruthy();
+    expect(getByTestId('feedback-widgetID')).toBeTruthy();
+  });
+
+  describe('ReactNativeProfiler', () => {
+    it('uses given options when set', () => {
```

Comment:
Since we removed the `updateProps` from the type, I think the option should be ignored.

Reply:
fixed on https://github.com/getsentry/sentry-react-native/pull/4740/commits/a38cbcb2b699583c1bcc8daf3a3efcac1aef825c